src/lib/libwasmfs.js

```diff
@@ -503,7 +503,7 @@ addToLibrary({
     return FS_mknod(path, mode, 0);
   },
 
-  $FS_writeFile__deps: ['_wasmfs_write_file', '$stackSave', '$stackRestore'],
+  $FS_writeFile__deps: ['_wasmfs_write_file', '$stackSave', '$stackRestore', 'malloc', 'free'],
```

Comment:
Is this related?   Maybe land this separately?

Reply:
https://github.com/emscripten-core/emscripten/pull/24036